src/api/IInput.js

```diff
@@ -56,5 +56,29 @@
     IInput.prototype.change = function (w) {
     };
 
+    IInput.prototype.checkNodeName = function (expected, element) {
+        var remove = this._inputElement.some(function(e) {
+            if (e.node().nodeName !== expected) {
+                return true;
+            }
+        });
+        if (remove) {
+            this._inputElement.forEach(function(e) {
+               e.node().remove();
+            });
+            this.createInput(element);
+        }
```

Comment:
i think that was the original concept yes ... if we dont need it we can remove... thoughts?